### ISSUE TITLE
By the way... why have you followed me?

### ISSUE THREAD

**Issue by verbose9** [7h3und34d/sexy-pokemon-filter#1]

**Comment by 7h3und34d** [comment]
We are friends in Discord. I think we met in Blue Collar Coder channel.

**Comment by verbose9** [comment]
> We are friends in Discord. I think we met in Blue Collar Coder channel.

Right. Well I'm a cryptocurrency guy and don't use typescript much. Do you wanna start a trade with me. Don't worry no credit cards required.
